QUESTIONS.md

```diff
@@ -3,3 +3,11 @@
 What questions can you come up with?
 
 ## Cohort Questions
+
+1. Is "Time-Travel" exclusive to React? And is it possible to Time-Travel back to week 1 with my current knowledge so I know what I'm in for and can mentally prepare?!
+
+2. Why is immutability important and is it's importance specific to React or is it as important for all of JS?
+
+3. Are React Devtools different than Chrome Devtools?
```

Comment:
Yes! It adds 2 new tabs to your Chrome dev tools. It's great for seeing props without having to console.log

Reply:
Awesome! I'll definitely look into it! Thanks!